ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+# Lapis Manalis
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# Albion #
+#~~~~~~~~#
+
+# -ii 7A7D
+
+0.0 "--sync--" sync / 00:0839::The Silvan Throne will be sealed off/ window 0,1
+6.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+10.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+21.6 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
```

Comment:
Oh wow, is this the most we get for this? Is there no ability or map effect?

Reply:
Each individual animal in the crossing uses `0x7A7D Wildlife Crossing`; you get a billion of them overlapping with 0.5 cast time differences, and there's no way to identify which one starts the first crossing vs the second.

Reply:
Oooof, ok!